app/src/main/AndroidManifest.xml

```diff
@@ -29,7 +29,7 @@
 
         <meta-data
             android:name="com.google.android.geo.API_KEY"
-            android:value="AIzaSyCDQ6tzZ9Azpba8ZAEysJ4VskXTnW4_LAo"/>
```

Comment:
If we wanted to be safe we should rewrite the history of the repo to remove the key or change it but not really important as it is a test